Jenkinsfile

```diff
@@ -761,12 +761,10 @@ def setupNode() {
 
 def getUniqueID(testId, useSha) {
   // This is used also for Kubernetes namespace and should not > 62 chars.
-  def date = new Date().format("yyMMddHHmm").toString()
-  def prefix = isReleaseQualification() ? 'rc-test' : 'test'
+  def prefix = isReleaseQualification() ? 'rc-' : ''
   def uuid = randomUUID().toString()
-  def sha = useSha ? "${GIT_SHA.take(7)}-" : ''
   def identifier = testId != '' ? "${testId}-" : ''
-  return "${prefix}-${sha}${identifier}${uuid.take(4)}-${date}"
+  return "${prefix}${identifier}${uuid.take(6)}"
```

Comment:
The maximum length of the service name is 63. I had to remove some field to reduce the length.

e2e.sh and e2e-kube.sh are creating a new service and remove it in a cleanup function.
Currently the e2e test creates and removes 12 services. 

When both e2e and pr-e2e jobs are running concurrently, then we could use maximum 24 services.

For now endpoints-jenkins project has 53 services which is not managed by e2e tests.
We need to remove services not in use.

https://pantheon.corp.google.com/endpoints?project=endpoints-jenkins&organizationId=433637338589